app/components/paperShow/index.tsx

```diff
@@ -815,11 +753,64 @@ class PaperShow extends React.PureComponent<PaperShowProps, PaperShowStates> {
 
   private fetchComments = (page: number = 1) => {
     const { paper, dispatch } = this.props;
-
     if (paper) {
       dispatch(getComments({ paperId: paper.id, page }));
     }
   };
+
+  private fetchPapersInCollection = (collectionId: number) => {
+    const { myCollections, dispatch } = this.props;
+    if (myCollections.length > 0) {
+      dispatch(getPapers(collectionId));
+    }
+  };
+
+  private getMyCollections = async () => {
+    const { dispatch, currentUser, paper } = this.props;
+    if (currentUser && currentUser.isLoggedIn && (currentUser.oauthLoggedIn || currentUser.emailVerified)) {
+      try {
+        const selected_collection_index = Number(Cookies.get(SELECTED_COLLECTION_INDEX)) || 0;
```

Comment:
`parseInt(Cookies.get(SELECTED_COLLECTION_INDEX) || 0, 10);` is safer.

Reply:
`selected_collection_index` is snake_case.
we're using camelCase. (selectedCollectionIndex is preferred)